app/(api)/_schema/collection_schema.mjs

```diff
@@ -0,0 +1,12 @@
+import mongoose from 'mongoose';
```

Comment:
we aren't using mongoose. Just follow the way we do other schema stuff. Mongoose does not alter the database itself, it's just an ORM.